### PR TITLE
Fabo/4762 fix extension hmr

### LINKED ISSUE #4762: Extension hot reload failing: FILE NOT FOUND
Probably just some silly webpack bug:

image

Steps to reproduce:

- build extension locally
- run `yarn watch:dev`
- make some change in the code and try to open extension again

You will get the above blank screen with the FILE NOT FOUND error.

To fix it you need to rerun `yarn watch:dev`

### PR DESCRIPTION
**Pull request by faboweb**
Closes #4762

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: extension/package.json [hunk 1]
```diff
@@ -13,13 +13,13 @@
     "lint:fix": "eslint --ext .js,.vue src --fix && stylelint \"src/**/*.{css,vue}\" --fix",
     "lint:js": "eslint --ext .js,.vue src",
     "lint:css": "stylelint \"src/**/*.{css,vue}\"",
-    "build": "node ./scripts/check-env-variables.js && cross-env NODE_ENV=production webpack --hide-modules && node ./scripts/remove-localhost.js",
-    "build:dev": "node ./scripts/check-env-variables.js && cross-env NODE_ENV=development webpack --hide-modules",
+    "build": "del dist && node ./scripts/check-env-variables.js && cross-env NODE_ENV=production webpack --hide-modules && node ./scripts/remove-localhost.js",
```
Comment (Bitcoinera) [inline extension/package.json:16]: what is this command? delete? Does this also work in Linux? (`rm -r`)

File: extension/package.json [hunk 2]
```diff
@@ -99,10 +99,10 @@
     "babel-jest": "^26.0.1",
     "babel-loader": "^8.0.2",
     "chromedriver": "^81.0.0",
-    "clean-webpack-plugin": "^3.0.0",
     "copy-webpack-plugin": "^4.5.3",
     "cross-env": "^5.2.0",
     "css-loader": "^3.5.3",
+    "del-cli": "^3.0.1",
```
Comment (Bitcoinera) [inline extension/package.json:105]: ah ok, it is a module. I see

**Comment by Bitcoinera** [comment]
Cool, will test this in a bit

**Review by Bitcoinera — APPROVED** [review]

File: extension/package.json (hunk 1 above)
Comment (faboweb) [reply extension/package.json:16]: this is an npm package to work cross platform

**Pull request merged**